app/models/document_capture_session.rb

```diff
@@ -80,6 +90,17 @@ def expired?
       Time.zone.now
   end
 
+  def reset
+    self.result_id = nil
+    now = Time.zone.now
+    self.created_at = now
+    self.updated_at = now
```

Comment:
- I feel like changing `created_at` can make things confusing. Are we looking for when "this session" started? If so, what if we added a column
- `updated_at` will automatically get written when we save, I don't think we need to have it here

Reply:
I was wondering if any logic checked when the latest session was created but probably not